addons/move-to-top-layer/addon.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name":"Move Sprite to Top Layer",
+  "description":"This addon allows you to move a selected sprite to the top layer",
+  "info": [
+    {
+      "type": "notice",
+      "text": "Press \"Shift\" while clicking a sprite in the sprite list to both select and move it to the top layer.",
```

Comment:
I think we could include this note in the addon's description instead of a notice.